<launch>

    <arg name="datasetfolder" default="$(find lips_matlab)/output/spencer_01/" />

    <node pkg="lips_simulator" type="pubSimulation" name="pubSimulation" output="screen" args="$(arg datasetfolder)">
        <!-- Gyroscope Noise -->
        <param name="gyroscope_noise_density"      type="double"  value="0"/>
        <param name="gyroscope_random_walk"        type="double"  value="0"/>
        <param name="gyroscope_bias_init"          type="double"  value="0"/>
        <!-- Accelerometer Noise -->
        <param name="accelerometer_noise_density"  type="double"  value="0"/>
        <param name="accelerometer_random_walk"    type="double"  value="0"/>
        <param name="accelerometer_bias_init"      type="double"  value="0"/>
        <!-- Lidar point cloud noise -->
        <param name="sigma_lidar_pts"              type="double"  value="0"/>
    </node>

</launch>
